### PR TITLE
Add script to regenerate test-certificate fixtures

### PR DESCRIPTION
**Pull request by thaJeztah**
follow-up to https://github.com/docker/cli/pull/2428

depends on https://github.com/dmcgowan/quicktls/pull/3

### CODE REVIEW

**Comment by thaJeztah** [comment]
Right now it's to manually update the certificates; we could generate them "on the fly" it we want to (still have to wait for the changes to be merged upstream) 😅

File: e2e/testdata/gen-certs.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+GO111MODULE=off go get -u github.com/dmcgowan/quicktls
+cd notary || exit 1
```
Comment (AkihiroSuda) [inline e2e/testdata/gen-certs.sh:4]: `set -eu` instead of `exit 1`?

Comment (thaJeztah) [reply e2e/testdata/gen-certs.sh:4]: Ah, yes, I think that works as well; this was mostly to satisfy the linter 😅

**Comment by thaJeztah** [comment]
Updated; PTAL

**Comment by thaJeztah** [comment]
bringing this one in

**Pull request merged**